app/components/ResumeCard.tsx

```
@@ -1,29 +1,92 @@
+            setResumeUrl(url);
+        }
+
+        loadResume();
+    }, [imagePath, isPreview]);
+
+    return (
+        <div 
+            className="resume-card hover:shadow-lg transition-all duration-300 hover:-translate-y-1"
+            onClick={isPreview ? auth.signIn : undefined}
+            style={isPreview ? {cursor: 'pointer'} : {}}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)